Artifacts/windows-firewall/Artifactfile.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+  "title": "Execute basic Windows Firewall operations",
+  "description": "Artifact to execute basic Windows firewall commands. This could be useful if you mistakenly enable rules which would deny access to your VM",
+  "publisher": "Milan Musec",
+  "tags": [
+      "Windows"
+  ],
+  "targetOsType": "Windows",
+  "parameters": {
+    "firewallOperation": {
+      "type": "string",
+      "displayName": "Firewall Operation:",
+      "description": "Select firewall operation",
+      "defaultValue": "-- none --",
+      "allowedValues": [
+          "-- none --",
+          "Firewall Status",
+          "Enable ICMP (ping)",
+          "Enable RDP Port (tcp/3389)",
```

Comment:
Note that this artifact alone will not be sufficient to enable ports like RDP. If the Lab VM is using a Shared IP configuration, a NAT rule is also required. So, users may be somewhat mislead that this is enough to make that work. I'm not sure how this can be communicated to the consumer of this artifact.

Reply:
Indeed, only what could help include text inside artifact which will be displayed in log after artifact is applied and explain